voice/voice_block.py

```diff
@@ -6,36 +7,41 @@
 except:
     # keeps this for backwards compatability
     from twilio import TwilioRestException
+from nio.common.signal.base import Signal
 from nio.common.block.base import Block
 from nio.common.discovery import Discoverable, DiscoverableType
 from nio.common.versioning.dependency import DependsOn
-from nio.metadata.properties.list import ListProperty
-from nio.metadata.properties.expression import ExpressionProperty
-from nio.metadata.properties.object import ObjectProperty
-from nio.metadata.properties.string import StringProperty
+from nio.metadata.properties import ExpressionProperty, IntProperty, \
+    ListProperty, ObjectProperty, StringProperty
 from nio.modules.web import WebEngine
 from nio.modules.web import RESTHandler
 from nio.modules.threading import Thread
 from nio.util.unique import Unique
-from blocks.twilio_blocks.sms.sms_block import Recipient, TwilioCreds
+from ..sms.sms_block import Recipient, TwilioCreds
 
 
 class Speak(RESTHandler):
 
-    def __init__(self, messages):
-        super().__init__('/')
+    def __init__(self, endpoint, notifier, logger, messages):
```

Comment:
Does it maybe make more sense to just pass a reference to the block here, then do something like `self.blk._logger` and `self.blk.notify_signals`? That would potentially be more "future proof"